Tensile/Source/TensileCreateLibrary.cmake

```diff
@@ -32,7 +32,13 @@ function(TensileCreateLibraryCmake
     Tensile_MERGE_FILES
     Tensile_SHORT_FILE_NAMES
     Tensile_LIBRARY_PRINT_DEBUG )
-
+# make Tensile_PACKAGE_LIBRARY and optional parameter 
```

Comment:
Can you update the new version of this function as well?

Reply:
Not sure what you are asking here is there another version of this function?

Reply:
The newer version is in [TensileConfig.cmake](https://github.com/ROCmSoftwarePlatform/Tensile/blob/a89467f8e7627782acf5c5e24e82095cea39aa9d/Tensile/cmake/TensileConfig.cmake#L68).